### PR TITLE
Add blog post for aurora 2023 updates

### PR DESCRIPTION
**Pull request by kara**
Adds blog post for Aurora's 2023 roadmap.

### CODE REVIEW

File: site/en/blog/aurora-update-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+* **Angular image directive and `nuxt/image` features**: We have a number of exciting features planned for Angular, such as automatic preconnect hint generation, migration tools to aid the transition from basic `<img>` tags, picture tag support, and placeholders. We will also be consulting with the Nuxt team on a number of new features for `nuxt/image`.
+
+* **INP research (cross-framework)**: As [Interaction to Next Paint (INP)](https://web.dev/inp/) will be replacing [First Input Delay (FID)](https://web.dev/fid/) in 2024, we are ramping up support for improving INP in frameworks. This will involve analysis of the root causes for INP issues in frameworks and creating built-in solutions for framework users where possible.
+
+* **Speedometer 3**: We are also helping to implement the next generation of the benchmarking tool [Speedometer](https://browserbench.org/Speedometer2.0/) to represent the modern web framework landscape of 2023. 
+
+## Stay up-to-date
```
Comment (malchata) [inline site/en/blog/aurora-update-2023/index.md:107]:
```suggestion
## Stay up to date
```

**Review by malchata — APPROVED** [review]
Some quick nits, nothing big. Once you've committed the suggested changes, merge whenever you're ready!

File: site/en/blog/aurora-update-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+For the past few years, we've been partnering with frameworks like Next.js, Angular, and Nuxt to optimize [Core Web Vitals](https://web.dev/vitals/).  Here are some highlights since our last update.
+
+### Images
+
+Images are often a source of performance issues. The following are some of the methods we've been pursuing with framework stakeholders to ensure that best practices are available out of the box, so that developers are delivering images optimally by default when they use the frameworks we're partnering with.
+
+#### Angular image directive
+
+We published a [stable image directive](https://angular.io/guide/image-directive) for the Angular framework, available in Angular 15 and specially backported to 13.4 and 14.3. This directive enables [native lazy loading](http://el-image-lazy-loading/) by default, adds [`fetchpriority` hints](https://web.dev/fetch-priority/) to priority images, and automatically generates appropriate `srcset` attributes for responsive images.
```
[the remark this answers is not in the document]
Comment (kara) [reply site/en/blog/aurora-update-2023/index.md:30]: Oh, thank you!

**Comment by kara** [comment]
@malchata Thanks for the quick review! Committed your suggestions. Would you mind merging it? I don't have merge access myself.

**Review by housseindjirdeh — APPROVED** [review]
This is great, thanks for writing this!

LGTM :shipit:

File: site/en/blog/aurora-update-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's New with Aurora?
+description: >
+  Learn what the Chrome Aurora team has been up to lately and their roadmap for 2023.
+date: 2023-05-16
```
Comment (malchata) [inline site/en/blog/aurora-update-2023/index.md:6]:
```suggestion
date: 2023-05-19
```

**Pull request merged**